app/templates/Gruntfile.js

```diff
@@ -133,7 +133,14 @@ module.exports = function (grunt) {
           livereload: true,
           nospawn: true //Without this option specified express won't be reloaded
         }
-      }
+      },
+      bower: {
+        files: ['bower.json'],
+        tasks: ['wiredep'],
+        options: {
+          livereload: true
```

Comment:
We don't really need the livereload hook since a change to `index.html` will trigger it.